src/custom/pages/Account/index.tsx

```diff
@@ -29,6 +34,23 @@ function _getPropsFromRoute(route: string) {
   }
 }
 
+// Note: As we build these single pages, we will remove this component in the future
+const Overview = () => {
+  const { chainId } = useWeb3React()
+
+  return (
+    <Container>
+      {chainId && chainId === ChainId.MAINNET && <AffiliateStatusCheck />}
```

Comment:
Nitpick: If chainId is undefined the check will fail against mainnet anyway

```suggestion
      {chainId === ChainId.MAINNET && <AffiliateStatusCheck />}
```